### DISCUSSION TITLE
Weird installation status strings

### DISCUSSION THREAD

**Discussion by rprueckl** [wixtoolset/issues#6789] [Questions] [answered]
Hi community,

not sure whether I am doing anything wrong or this is a bug - I see weird status strings while installing my package on Windows 11. I do not see them when installing on Windows 10.

Here is an example:
![InkedMicrosoftTeams-image (3)_LI](https://user-images.githubusercontent.com/53342897/170971600-df33e14f-8b2e-4185-af4b-1a7a711d3e97.jpg)
The unexpected addition of the status here is "File: [1]"

I am using WiX 3.11.2.

Thanks in advance for your help.

**Comment by robmen** [comment]
Definitely seems wrong but the Windows Installer team owns rendering that content so you'll want to follow up with them at Microsoft. IIRC, this is not the first time this problem has happened.

**Reply by rprueckl** [reply]
> See accepted answer at https://stackoverflow.com/questions/44161526/wix-copying-new-files-file-1-directory-9-size-6-shown-during-instal - add a reference to "WixUI_ErrorProgressText".

js69 - this is the answer - please post as answer and it can be accepted

**Comment by js69** [comment, marked as answer]
See accepted answer at https://stackoverflow.com/questions/44161526/wix-copying-new-files-file-1-directory-9-size-6-shown-during-instal - add a reference to "WixUI_ErrorProgressText".

**Reply by robmen** [reply]
The root issue is still a bug in the Windows Installer. The `Error` table should not be required but providing one is a good workaround.
